larq/callbacks_test.py

```diff
@@ -1,7 +1,14 @@
 import numpy as np
 import tensorflow as tf
 import larq as lq
+
+
+from larq import testing_utils as lq_testing_utils
+from larq.callbacks import BopLearningRateScheduler
+
+from tensorflow.python import keras
```

Comment:
I think using `tf.keras` directly or `from tensorflow import keras` would be better since `tensorflow.pyhon` is a private API.

Reply:
ok, changed it.